examples/gno.land/p/demo/gnolotto/gnolotto.gno

```diff
@@ -0,0 +1,113 @@
+package gnolotto
```

Comment:
Can you add a short godoc comment about the package to give a brief intro as to what it's about, what it can be used for?